src/components/HistoricalWeatherWidget/HistoricalWeatherWidget.tsx

```diff
@@ -0,0 +1,112 @@
+import { useEffect, useState } from 'react';
+import { format } from 'date-fns';
+import pick from 'lodash/pick';
+import CircularProgress from '@mui/material/CircularProgress';
+import { useSnackbar } from 'notistack';
+import { FormValuesType } from 'components/HistoricalWeatherForm/validation';
+import TableCell from 'components/TableCell';
+import ForecastBody from 'types/forecast';
+import findCountryNameByCode from 'utils/findCountryNameByCode';
+import FORECAST from './forecast';
+import WEEK_DAYS from './weekDays';
+
+type Forecast = ForecastBody[];
+
+type WeeklyForecast = Array<ForecastBody[]>;
+
+interface Props {
+  formValues: FormValuesType;
+}
+
+const SPINNER_SIZE = 25;
+const WEEK_LENGTH = 7;
+
+export default function HistoricalWeatherWidget({ formValues }: Props) {
+  const { enqueueSnackbar } = useSnackbar();
+  const [forecast, setForecast] = useState<Forecast | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const weeklyForecast = [[]];
+
+  useEffect(() => {
+    const fetchForecast = () => {
+      try {
+        setIsLoading(true);
+        const { list } = FORECAST;
```

Comment:
Don't desctructure fixtures. Usually props from them extracted without destructuring in order to make search of variable easier